package.json

```diff
@@ -25,15 +25,17 @@
     "registry": "https://registry.npmjs.org/"
   },
   "devDependencies": {
-    "@ephox/bedrock-client": "^9.3.2",
-    "@ephox/bedrock-server": "^9.3.2",
+    "@ephox/bedrock-client": "^11.1.1",
+    "@ephox/bedrock-server": "^11.2.0",
+    "@tinymce/beehive-flow": "^0.14.0",
     "@tinymce/eslint-plugin": "^1.7.1",
     "awesome-typescript-loader": "^5.2.0",
-    "source-map-loader": "^1.1.1",
+    "source-map-loader": "^2.0.1",
     "typescript": "^4.0.3",
-    "webpack": "^4.25.1"
+    "webpack": "^5.24.4"
```

Comment:
Does this actually work? As when we tried last time webpack 5 wouldn't work with the configuration we've used in most projects.

Reply:
I've redone the dependency updates so we only do a minor update of webpack.